### PR TITLE
Slightly reword of the README

### PR DESCRIPTION
**Pull request by mcollina**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,12 +42,31 @@ fastify.listen(3000, function (err) {
 })
 ```
 
+with async-await:
+
+```js
+// Require the framework and instantiate it
```
Comment (delvedor) [inline README.md]: I think we can remove the all comments :)

File: README.md [hunk 2]
```diff
@@ -43,12 +42,31 @@ fastify.listen(3000, function (err) {
...
+const fastify = require('fastify')()
+
+// Declare a route
+fastify.get('/', async (request, reply) => {
+  reply.header('Content-Type', 'application/json').code(200)
```
Comment (allevo) [inline README.md]: `type`?

File: README.md [hunk 3]
```diff
@@ -43,12 +42,31 @@ fastify.listen(3000, function (err) {
...
 
+with async-await:
+
+```js
+// Require the framework and instantiate it
+const fastify = require('fastify')()
+
+// Declare a route
+fastify.get('/', async (request, reply) => {
+  reply.type('application/json').code(200)
```
Comment (delvedor) [inline README.md]: Actually this is not needed, since `application/json` and `200` are our defaults :D

**Comment by mcollina** [comment]
Updated

**Comment by delvedor** [comment]
I'll add also a note on our plugin architecture, its flexibility and extensibility is a selling point.

*Enter Fastify. Fastify is a web framework highly focused on providing the best developer experience with the least overhead and a powerful plugin architecture. It is inspired by Hapi and Express and as far as we know, it is one of the fastest web frameworks in town*

File: README.md [hunk 4]
```diff
@@ -43,12 +42,28 @@ fastify.listen(3000, function (err) {
...
+})
+```
+
 Do you want to know more? Head to the <a href="https://github.com/fastify/fastify/blob/master/docs/Getting-Started.md"><code><b>Getting Started</b></code></a>.
 
 ### Core features
 
 - **100% asynchronous:** all the core is implemented with asynchronous code, in this way not even a millisecond is wasted.
-- **Highly performant:** as far as we know, Fastify is one of the fastest web frameworks in town, depending on the code complexity we can serve up to 20000 request per second.
+- **Highly performant:** as far as we know, Fastify is one of the fastest web frameworks in town, depending on the code complexity we can serve up to 34000 request per second.
```
Comment (jsumners) [inline README.md]: `34000 request per second` => `34000 requests per second`

**Comment by mcollina** [comment]
@jsumners updated

**Pull request merged**